tox.ini

```diff
@@ -1,69 +1,59 @@
 [tox]
-envlist = py38, pylint, format, flake8, docs
+envlist = py38, py39, py310, pylint, format, flake8, docs
 skip_missing_interpreters = true
+isolated_build = true
 
 [testenv]
-changedir = tests
```

Comment:
Why don't we want to do this anymore?

Reply:
I added tests/ to the end of the pytest commands.  When tox changes directories, it apparently doesn't go back so if you run multiple with say `tox -p all`; the second one will fail because its trying to run in the tests folder!